### PR TITLE
Tour Rework Changes

### PR DESCRIPTION
**Pull request by LizGoro90**

### CODE REVIEW

File: tutorials/transactions/01/README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
-# How to use transactions
+### Transactions
```
Comment (LizGoro90) [inline tutorials/transactions/01/README.md]: Section must be included in step "Insert, Update, and Delete Items in a Result Set".
screen shot 2018-08-09 at 7 10 48 pm

File: tutorials/sql-builder/02/README.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
-# JOIN queries and struct composition
+### JOIN Queries and Struct Composition
```
Comment (LizGoro90) [inline tutorials/sql-builder/02/README.md]: Section must be placed under "SQL Builder" in step "Build Common Queries".
screen shot 2018-08-10 at 8 25 46 am

File: tutorials/sql-builder/04/README.md [hunk 3]
```diff
@@ -1,23 +1,20 @@
-# Using raw SQL
+## b) Raw SQL
```
Comment (LizGoro90) [inline tutorials/sql-builder/04/README.md]: Section must be included in step "Build Common Queries".
screen shot 2018-08-10 at 8 30 22 am

File: tutorials/sql-builder/03/README.md [hunk 4]
```diff
@@ -1,17 +1,4 @@
...
-```go
-var eaPoe Author
-
-q := sess.SelectFrom("authors").
-  Where("last_name", "Poe").
-  One(&eaPoe)
-
-err = q.One(&eaPoe)
-```
+## a) SQL Builder
```
Comment (LizGoro90) [inline tutorials/sql-builder/03/README.md]: Section must be included in step "Update, Insert, or Delete Items in a Result Set".
screen shot 2018-08-10 at 8 34 04 am

File: tutorials/queries/06/README.md [hunk 5]
```diff
@@ -1,9 +1,8 @@
-# Updating or deleting items on a result set
+# Update, Insert, or Delete Items in a Result Set
```
Comment (LizGoro90) [inline tutorials/queries/06/README.md]: Remember that the Insert method is missing in this section. An example must be included.

File: tutorials/sql-builder/03/README.md [hunk 6]
```diff
@@ -66,10 +53,23 @@ q := sess.DeleteFrom("books").
...
-See the
-[sqlbuilder.SQLBuilder](https://godoc.org/upper.io/db.v3/lib/sqlbuilder#SQLBuilder)
-interface to learn about all available methods that can be used to build or
-execute SQL statements.
+![Note](https://github.com/LizGoro90/db-tour/tree/master/static/img)
+> Take a look at the [sqlbuilder.SQLBuilder](https://godoc.org/upper.io/db.v3/lib/sqlbuilder#SQLBuilder) interface to learn about all available methods for building
+and executing SQL statements.
+
+
+## b) Raw SQL
```
Comment (LizGoro90) [inline tutorials/sql-builder/03/README.md]: Remember that the Insert and Delete methods are missing in this section. An example must be included.

File: tutorials/queries/01/main.go [hunk 7]
```diff
@@ -37,17 +34,4 @@ func main() {
 
 	booksTable := sess.Collection("books")
 
-	res := booksTable.Find().OrderBy("id")
```
Comment (LizGoro90) [inline tutorials/queries/01/main.go]: This section must be restored so the example code remains interactive.

**Comment by xiam** [comment]
Thanks a lot @LizGoro90

**Pull request merged**